<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guilano Aurora AI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>481</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="monitor_tab">
     <attribute name="title">
      <string>Monitor</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>127</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_ip">
           <property name="text">
            <string>IP :</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>28</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ip">
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_port">
           <property name="text">
            <string>Port :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_port">
           <property name="text">
            <string>9999</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>120</width>
             <height>34</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_disconnect">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="configure_tab">
     <attribute name="title">
      <string>Configure</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="outdoor_tab">
     <attribute name="title">
      <string>Outdoor</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add_wayPoint">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add WayPoint</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_remove_wayPoint">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove WayPoint</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_send_wayPoint">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send WayPoints</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>261</width>
        <height>143</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_guided_mode">
         <property name="text">
          <string>Guided Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_arm">
         <property name="text">
          <string>Arm</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_altitude">
           <property name="text">
            <string>Altitude :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_altitude">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_takeoff">
           <property name="text">
            <string>TakeOff</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_land_mode">
         <property name="text">
          <string>Land Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_rtl_mode">
         <property name="text">
          <string>RTL Mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>210</y>
        <width>241</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Mavros/state:</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>221</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_connected">
            <property name="text">
             <string>Connected  :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_armed">
            <property name="text">
             <string>Armed :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_guided">
            <property name="text">
             <string>Guided :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_mode">
            <property name="text">
             <string>Mode :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_show_connected">
            <property name="text">
             <string>---</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_show_armed">
            <property name="text">
             <string>---</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_show_guided">
            <property name="text">
             <string>---</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_show_mode">
            <property name="text">
             <string>---</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_start_reading">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Start reading</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stop_reading">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop reading</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_start_mission">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Mission</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_read_wayPoint">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>250</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Read WayPoint</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="map_tab">
     <attribute name="title">
      <string>Map</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_latitude">
             <property name="text">
              <string>Latitude :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_longitude">
             <property name="text">
              <string>Longitude :</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="lineEdit_latitude">
             <property name="text">
              <string>-35.3632622</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_longitude">
             <property name="text">
              <string>149.1652374</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Zoom :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Blocks :</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLineEdit" name="lineEdit_zoom">
             <property name="text">
              <string>21</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_blocks">
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>189</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_set_default_tile">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Default Tile</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_change_map_path">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Map Path</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_generate_tile">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Tile</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_read_takeoff_gps">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Read Takeoff GPS</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
